internal/storage/db.go

```diff
@@ -21,11 +21,21 @@ const assignTokensQuery = `
 UPDATE
 	rewards_api.rewards
 SET
-	tokens =
-		div(
-			(streak_points + integration_points) * $2::numeric,
-			(SELECT sum(streak_points + integration_points) FROM rewards_api.rewards WHERE issuance_week_id = $1)
-		)
+  streak_tokens =
```

Comment:
Can you format this the way the rest of it is? I think I used https://sqlformat.darold.net